h2o-docs/src/product/welcome.rst

```diff
@@ -893,3 +893,123 @@ After obtaining the IP address, point your browser to the specified ip address a
         # Initialize H2O 
         import h2o
         docker_h2o = h2o.init(ip = "192.168.59.103", port = 54321) 
+
+
+Kubernetes Integration
+----------------------
+
```

Comment:
```suggestion
H2O nodes must be treated as stateful by Kubernetes environment.  In Kubernetes, a set of pods sharing a common state is named as StatefulSet. H2O is a stateful application. H2O Nodes are spawned together and deallocated together, as a single unit. Kubernetes tooling for stateless applications in not applicable to H2O.
```

Reply:
One small suggestion, I'm not sure it is necessary, as this concept is explained a little bit later. Feel free to discard.